js/src/personal/maliha/Maliha.page.tsx

```diff
@@ -1,17 +1,24 @@
 import { Title, Text } from '@mantine/core'
+import { useEffect } from 'react'
+import { Sandwich } from './Sandwich.tsx'
 import styles from './Maliha.module.css'
-import { Menu } from './Menu.tsx'
 
 export function MalihaPage() {
+  useEffect(() => {
+    document.body.style.backgroundColor = '#B07251'
```

Comment:
You shouldn't need to update this directly like this. There should be other ways of updating the color.

This is a very not-react way of doing things.